### ISSUE TITLE
Commands are not recognized unless registered in providers property

### ISSUE THREAD

**Issue by vorban** [squareboat/nest-console#10]
Hi! 

## Steps to reproduce

I followed the instructions in the README.md: 
- installed the package on an almost fresh Nest.js project
- I didn't touch the `cli` file as I'm following the standard structure
- Registered `ConsoleModule` in the `AppModule` imports property
```js
// ... more imports
import { ConsoleModule } from '@squareboat/nest-console';

@Module({
    imports: [
        ConsoleModule,
        // ... more module declarations
```
- Created a command file:
```js
// src/user/create-user.command.ts

import { Injectable } from '@nestjs/common';
import { Command } from '@squareboat/nest-console';

@Injectable()
@Command('user:create', {
    desc: 'Create a user with email and password',
})
export class CreateUser {
    handle() {
        console.log('IT WORKS!');
    }
}
```
- Ran `npm run build`

## Expected behaviour

Based purely on the `README.md` documentation, `node cli user:create` should work.

## Observed behaviour

```
$ node cli user:create
user:create : command not found
```

## How to fix

It works if I register my command in a module (here the nearest is `UserModule` so I put it there):
```js
import { Module } from '@nestjs/common';
import { CreateUser } from './create-user.command';
import { UserService } from './user.service';

@Module({
    providers: [UserService, CreateUser],
    exports: [UserService],
})
export class UserModule {}
```

This is what is done in your [nest-boilerplate](https://github.com/squareboat/nestjs-boilerplate) and that's how I figured it out.

## Conclusion

If this is expected behaviour, I strongly suggest adding this info to your documentation.

Currently, as read directly, it is explicitly stated that `whatever command you create in your application will be discovered automatically.` and that is not the case.

Thanks for the package nonetheless, CLI commands are a must-have 😃

**Comment by sb-vinayak** [comment]
This is resolved in v1.0.0

**Comment by vinayak25** [comment]
This is resolved in v1.0.0
